README.md

```diff
@@ -23,7 +23,7 @@ Setup and activate a Python virtualenv
 
 Install
 
-    pip install -e .[dev]
```

Comment:
did you intentionally commit this change in?

Reply:
Yeah, the command only worked for me with `pip install -e .` so I thought I'd pass it on. Not sure what the `[dev] part is about, but that may just be me being a noob in python

Reply:
hmm. i'm actually not positive what [dev] does. @mikeshultz added it so he may be able to provide an intelligent explanation.... :)

Reply:
`pip install -e .[dev]` installs the `dev` extras, which include everything necessary for development... 

https://github.com/OriginProtocol/origin-website/blob/d43a0127795f7e72974fb43fd014b6d52b5c27b2/setup.py#L45-L52

This is stuff not necessary for deploy so was sectioned out.

Reply:
I changed the README back.  If you don't need to testing suite, cool.  If you want a hand figuring out your problem, let me know.

This is pretty common Python packaging stuff, so no reason it shouldn't work.